lib/routes/account.js

```diff
@@ -390,6 +395,26 @@ module.exports = function (
             )
         }
 
+        function checkNumberOfActiveSessions () {
+          return db.sessions(emailRecord.uid)
```

Comment:
(Although the fact that this wasn't obvious, seems like a legitimate criticism of the way the code is currently structured)